graphql/schema/query.graphql

```diff
@@ -94,6 +94,9 @@ type Query {
   hasVersion(patchId: String!): Boolean!
   version(versionId: String! @requireProjectAccess(permission: TASKS, access: VIEW)): Version!
 
+  # image
+  image(imageId: String!): Image
+  
   # images
   images: [String!]!
```

Comment:
it's probably okay to just group these two together under the same comment since they're the same thematically